.php-cs-fixer.dist.php

```diff
@@ -8,6 +8,8 @@
     __DIR__.'/controllers',
     __DIR__.'/tests',
     __DIR__.'/tools/profiling',
+])->exclude([
+    'Unit/Resources',
```

Comment:
Needed.
If PHP CS Fixer analyzes this folder it will reject the PR because sample PHP files use `array()` instead of `[]`.

However PhpParameters rely on var_export that outputs `array()` and not `[]` so we have to keep `array()` for tests to validate the behavior.

Reply:
It's a bad practice to ignore a full directory.

You should ignore parts of the file : https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file

Reply:
Not sure it's a good idea to exclude this directory :thinking:

Reply:
@PierreRambaud 
I cannot modify the content of Unit/Resources folder because the php files need to be _exactly_ what the code I am working with will output.

So I cannot use php-cs-fixer comments like `// phpcs:disable` or `// phpcs:ignore`. If I add them my sample php file does not match anymore PrestaShop output.

As `exclude()` expects folders and not files, I cannot use it either to ignore only my sample files.

I searched whether we could disable the rule `Generic.Arrays.DisallowLongArraySyntaxSniff` for only a few files or one folder, but it seems [this is something not implemented in PHP CS Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3871).

I see no ways to only ignore files or to only disable folder scan for a single rule 🤷‍♂️  🤔

Reply:
To try: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1870/files